RabbitMQ.Stream.Client/HeartBeatHandler.cs

```diff
@@ -62,15 +67,15 @@ private async Task PerformHeartBeatAsync()
 
         // missed the Heartbeat 
         Interlocked.Increment(ref _missedHeartbeat);
-        LogEventSource.Log.LogWarning($"Heartbeat missed: {_missedHeartbeat}");
+        _logger?.LogWarning($"Heartbeat missed: {_missedHeartbeat}");
         if (_missedHeartbeat <= 3)
         {
             return;
         }
 
         // When the client does not receive the Heartbeat for three times the 
         // client will be closed
-        LogEventSource.Log.LogError($"Too many Heartbeat missed: {_missedHeartbeat}");
+        _logger?.LogError($"Too many Heartbeat missed: {_missedHeartbeat}");
```

Comment:
Same comment as above. I suggest `_logger?.LogError($"Too many heartbeats missed! ({MissedHeartBeatCount})", _missedHeartbeat);`

Reply:
Feel free to make changes!